src/main/java/net/projectzombie/region_rotation/file/FileRead.java

```diff
@@ -0,0 +1,138 @@
+package net.projectzombie.region_rotation.file;
+
+import net.projectzombie.region_rotation.modules.BaseState;
+import org.bukkit.Bukkit;
+import org.bukkit.World;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+
+/**
+ * Used to read in BaseStates.
+ * @author Gephery
+ */
+public class FileRead
```

Comment:
For 'static' classes like this, you need to make the constructor private and blank, i.e. no params, no internals, just an empty function. By default constructors are 'protected' or `public`.